core/src/main/scala/org/bitcoins/core/api/keymanager/KeyManagerApi.scala

```diff
@@ -0,0 +1,15 @@
+package org.bitcoins.core.api.keymanager
+
+import org.bitcoins.core.crypto.ExtPublicKey
+import org.bitcoins.core.hd.{HDAccount, HDPath}
+import org.bitcoins.crypto.Sign
+
+import scala.util.Try
+
+sealed trait KeyManagerApi
```

Comment:
I don't think we want to seal this, as other users may want to extend this to create their own API.